src/test/java/no/nav/fo/veilarbdialog/rest/DialogRessursTest.java

```diff
@@ -70,6 +75,25 @@ public void sistOppdatert() {
 
     }
 
+    @Test
+    public void nyHenvendelse() {
+
+        NyHenvendelseDTO dto = new NyHenvendelseDTO()
+                .setTekst("tekst");
+        String s =
+        given()
+                .port(port)
+                .contentType(ContentType.JSON)
+                .body(dto)
+                .post("/veilarbdialog/api/dialog?aktorId={aktorId}", "1337")
+                .then()
+                .extract()
+                .body()
+                .asString();
+        System.out.println(s);
```

Comment:
Forstår ikke helt hva denne tester? Hjelper meg forsåvidt med å se hva jeg har gjort feil med testen min lokalt(holder på med noe av det samme). Burde kanskje være komplett før den merges.

Reply:
Takk, her har jeg glemt av en ufullstendig test. Fullfører.

Bruker for øvrig det mønsteret tidlig i testen for å spare tid når jeg ikke er 100% på hva et gyldig kall forventes å skulle returnere. Et annet mønster er for øvrig noe som `.extract().body().as(DialogDTO.class)` og så gjøre assertions på objektet, dersom det blir for messy å bruke JSONPath.